src/uct/ib/rc/accel/rc_mlx5_ep.c

```diff
@@ -535,6 +535,35 @@ ucs_status_t uct_rc_mlx5_ep_fence(uct_ep_h tl_ep, unsigned flags)
     return uct_rc_ep_fence(tl_ep, &ep->tx.wq.fi, 1);
 }
 
+static ucs_status_t
+uct_rc_mlx5_ep_flush_check(uct_rc_mlx5_ep_t *ep, unsigned flags,
+                           uct_completion_t *comp)
+{
+    uct_rc_mlx5_iface_common_t *iface = ucs_derived_of(ep->super.super.super.iface,
+                                                       uct_rc_mlx5_iface_common_t);
+    uint64_t dummy; /* use this variable as dummy buffer to suppress
```

Comment:
what is the warning? We can pass NULL in `uct_rc_mlx5_ep_flush`

Reply:
```
In file included from /scrap/jenkins/workspace/ucx-3/contrib/../src/uct/ib/rc/accel/rc_mlx5.inl:11,

[2020-08-03T17:38:22.119Z]                  from /scrap/jenkins/workspace/ucx-3/contrib/../src/uct/ib/rc/accel/rc_mlx5_ep.c:22:

[2020-08-03T17:38:22.119Z] In function ‘uct_ib_mlx5_inline_copy’,

[2020-08-03T17:38:22.119Z]     inlined from ‘uct_rc_mlx5_txqp_inline_post’ at /scrap/jenkins/workspace/ucx-3/contrib/../src/uct/ib/rc/accel/rc_mlx5.inl:572:9,

[2020-08-03T17:38:22.119Z]     inlined from ‘uct_rc_mlx5_ep_flush_keepalive’ at /scrap/jenkins/workspace/ucx-3/contrib/../src/uct/ib/rc/accel/rc_mlx5_ep.c:557:5:

[2020-08-03T17:38:22.119Z] /scrap/jenkins/workspace/ucx-3/contrib/../src/uct/ib/mlx5/ib_mlx5.inl:172:9: error: argument 2 null where non-null expected [-Werror=nonnull]

[2020-08-03T17:38:22.119Z]   172 |         memcpy(dest, src, n);

[2020-08-03T17:38:22.119Z]       |         ^~~~~~~~~~~~~~~~~~~~
```